### PR TITLE
Toolbar: add 'kebabLimit' option.

### PR DESCRIPTION
**Pull request by martinpovolny**
Added option to configure the "kebabization" of the menu items.

 * kebabLimit === -1 means no compacting
 * kebabLimit === 0 means always compact into kebab
 * other values of kebabLimit give number of items to keep, before the rest is kebabized.

### TODO

 * ~~update specs~~
 * ~~add specs for the new behavior~~

### Testing

Running the `npm run server` gives you an example of behavior for different values of `kebabLimit`.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1683703

customb-2019-12-11_08-58

### Further changes:

 * update tests to make them pass (unrelated),
 * make `count` optional the same as `kebabLimit` (based on PR review feedback)

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@martinpovolny can you update the snapshot please?

File: demo/demo-app.jsx [hunk 1]
```diff
@@ -10,12 +10,46 @@ const wrapperComponent = () => (
   <React.Fragment>
     <div className="toolbar-pf row">
       <Toolbar
+        count={1}
```
Comment (Hyperkid123) [inline demo/demo-app.jsx]: Can I ask what does the count does and why is it required?

Comment (martinpovolny) [reply demo/demo-app.jsx]: In manageiq the toolbar needs count of selected items in the GTL on the page.

Anyway: making it optional, adding default value.

File: src/toolbar/Toolbar.jsx [hunk 2]
```diff
@@ -91,17 +107,23 @@ export const Toolbar = props => (
         .filter(toolbarGroupHasContent)
         .map((group, index) =>
           /* eslint react/no-array-index-key: "off" */
-          <ToolbarGroup key={index} onClick={props.onClick} group={collapseCustomGroups(group)} />)
+          <ToolbarGroup key={index} onClick={props.onClick} group={collapseCustomGroups(group, props.kebabLimit)} />)
```
Comment (Hyperkid123) [inline src/toolbar/Toolbar.jsx]: I suppose we don't have anything better than index right?

Comment (martinpovolny) [reply src/toolbar/Toolbar.jsx]: Not for the group. But the index is stable.

**Pull request merged**